src/scores/categorical/contingency_impl.py

```diff
@@ -727,7 +727,8 @@ def precision(self) -> xr.DataArray:
 
     def positive_predictive_value(self) -> xr.DataArray:
         """
-        Identical to :py:func:`success_ratio` and :py:func:`precision`.
+
+        Identical to :py:func:`success_ratio` and :py:func:`precision`
```

Comment:
```suggestion
        Identical to :py:func:`success_ratio` and :py:func:`precision`.
```
Suggesting to remove the empty line and add in a full stop (I think they may have accidentally crept in during a rebase).